docs/src/components/Layout/Sidebar.tsx

```diff
@@ -237,29 +239,22 @@ export const Sidebar = ({ expanded, setExpanded, platform }) => {
       <div className="docs-sidebar-overlay" onClick={onClick} />
       <div className="docs-sidebar-inner">
         <Flex direction="column" className="docs-sidebar-nav">
-          <Flex
-            direction="row"
-            alignItems="center"
-            justifyContent="space-between"
-          >
-            <LogoLink platform={platform} onClick={onClick} />
-            <MenuButton expanded={expanded} setExpanded={setExpanded} />
-          </Flex>
+          {hiddenOnDesktop ? (
+            <Flex
+              direction="row"
+              alignItems="center"
+              justifyContent="space-between"
+            >
+              <LogoLink platform={platform} onClick={onClick} />
+              <MenuButton expanded={expanded} setExpanded={setExpanded} />
+            </Flex>
+          ) : null}
```

Comment:
@dbanksdesign, I think the logo is showing up on mobile by design? Would it be okay to just remove it completely so that it matches desktop?